sys-apps/edac-utils/edac-utils-0.18-r1.ebuild

```diff
@@ -19,9 +19,18 @@ RDEPEND="${DEPEND}
 	sys-apps/dmidecode"
 
 src_prepare() {
+	default
+
+	# <GCC-10, <binutils-2.34 do not support LDPT_ADD_SYMBOLS_V2 https://sourceware.org/bugzilla/show_bug.cgi?id=25355
```

Comment:
Pop the link on a new line. I would also like a gentoo bug about this we can reference in the commit message?

Reply:
Maybe you could even fix the calls [CC](https://bugs.gentoo.org/725540) directly bug ;)